### ISSUE TITLE
Syntaxnet: 14 out of 52 bazel tests return errors after build

### ISSUE THREAD

**Issue by SaintNazaire** [tensorflow/models#2677]
**Issue**

Followed very exactly [Syntaxnet installation](https://github.com/tensorflow/models/tree/master/research/syntaxnet) to build syntaxnet into a docker container. Triggering the build unleashes a deluge of warnings. 38 tests out of 52 only complete without error. 14 tests fail.

This is a follow-up issue after fixing the memory issues that prevented compilation as described in #2588 and #2662 

**System information**

- Windows 10 professional
- Docker image based on Ubuntu 16.04.3 LTS
- Docker 17.09.0-ce
- Python version 2.7
- Bazel version 0.7.0
- CUDA/cuDNN version not relevant (CPU install)
- GPU model and memory not relevant (CPU install)

**Command triggering issue**

`cd /models/research/syntaxnet && test ...  --genrule_strategy=standalone --spawn_strategy=standalone`

**Top-level directory of the model you are using**

> /models/research/syntaxnet

**Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**

Followed very exactly  [Syntaxnet installation](https://github.com/tensorflow/models/tree/master/research/syntaxnet) procedure.

Only variation I made was to clearly specify environment variables:

```
ENV PYTHON_BIN_PATH=/usr/bin/python
ENV PYTHON_LIB_PATH=/usr/lib/python2.7
```
In addition added sandboxing exception handling:

`cd /models/research/syntaxnet && bazel test ... --genrule_strategy=standalone --spawn_strategy=standalone`

**Describe the problem**

38 tests out of 52 only complete without error.
14 tests fail.

**Error logs**

`bazel test ... --test_verbose_timeout_warnings`

> WARNING: /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/external/org_tensorflow/tensorflow/core/BUILD:1782:1: in includes attribute of cc_library rule @org_tensorflow//tensorflow/core:framework_headers_lib: '../../../../external/nsync/public' resolves to 'external/nsync/public' not below the relative path of its package 'external/org_tensorflow/tensorflow/core'. This will be an error in the future. Since this rule was created by the macro 'cc_header_only_library', the error might have been caused by the macro implementation in /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/external/org_tensorflow/tensorflow/tensorflow.bzl:1048:30
> WARNING: /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/external/org_tensorflow/tensorflow/contrib/learn/BUILD:15:1: in py_library rule @org_tensorflow//tensorflow/contrib/learn:learn: target '@org_tensorflow//tensorflow/contrib/learn:learn' depends on deprecated target '@org_tensorflow//tensorflow/contrib/session_bundle:exporter': No longer supported. Switch to SavedModel immediately.
> WARNING: /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/external/org_tensorflow/tensorflow/contrib/learn/BUILD:15:1: in py_library rule @org_tensorflow//tensorflow/contrib/learn:learn: target '@org_tensorflow//tensorflow/contrib/learn:learn' depends on deprecated target '@org_tensorflow//tensorflow/contrib/session_bundle:gc': No longer supported. Switch to SavedModel immediately.
> INFO: Analysed 218 targets (0 packages loaded).
> INFO: Found 166 targets and 52 test targets...
> FAIL: //syntaxnet:text_formats_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/text_formats_test/test.log)
> FAIL: //dragnn/python:network_units_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/network_units_test/test.log)
> FAIL: //dragnn/python:bulk_component_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/bulk_component_test/test.log)
> FAIL: //syntaxnet:graph_builder_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/graph_builder_test/test.log)
> FAIL: //examples/dragnn:test_run_all_tutorials (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/examples/dragnn/test_run_all_tutorials/test.log)
> FAIL: //syntaxnet:reader_ops_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/reader_ops_test/test.log)
> FAIL: //dragnn/python:spec_builder_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/spec_builder_test/test.log)
> FAIL: //dragnn/tools:model_trainer_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/tools/model_trainer_test/test.log)
> FAIL: //dragnn/python:sentence_io_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/sentence_io_test/test.log)
> FAIL: //dragnn/python:graph_builder_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/graph_builder_test/test.log)
> FAIL: //syntaxnet:parser_trainer_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/parser_trainer_test/test.log)
> FAIL: //syntaxnet:lexicon_builder_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/lexicon_builder_test/test.log)
> FAIL: //syntaxnet:beam_reader_ops_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/beam_reader_ops_test/test.log)
> FAIL: //dragnn/python:lexicon_test (see /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/lexicon_test/test.log)
> INFO: Elapsed time: 7.235s, Critical Path: 1.88s
> INFO: Build completed, 14 tests FAILED, 15 total actions
> //dragnn/components/stateless:stateless_component_test          (cached) PASSED in 0.2s
>   WARNING: //dragnn/components/stateless:stateless_component_test: Test execution time (0.2s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/components/syntaxnet:syntaxnet_component_test          (cached) PASSED in 0.1s
>   WARNING: //dragnn/components/syntaxnet:syntaxnet_component_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/components/syntaxnet:syntaxnet_link_feature_extractor_test (cached) PASSED in 0.1s
>   WARNING: //dragnn/components/syntaxnet:syntaxnet_link_feature_extractor_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/components/syntaxnet:syntaxnet_transition_state_test   (cached) PASSED in 0.1s
>   WARNING: //dragnn/components/syntaxnet:syntaxnet_transition_state_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/core:beam_test                                         (cached) PASSED in 0.1s
>   WARNING: //dragnn/core:beam_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/core:compute_session_impl_test                         (cached) PASSED in 0.1s
>   WARNING: //dragnn/core:compute_session_impl_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/core:compute_session_pool_test                         (cached) PASSED in 0.2s
>   WARNING: //dragnn/core:compute_session_pool_test: Test execution time (0.2s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/core:dragnn_bulk_op_kernels_test                       (cached) PASSED in 0.1s
>   WARNING: //dragnn/core:dragnn_bulk_op_kernels_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/core:dragnn_op_kernels_test                            (cached) PASSED in 0.1s
>   WARNING: //dragnn/core:dragnn_op_kernels_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/core:index_translator_test                             (cached) PASSED in 0.1s
>   WARNING: //dragnn/core:index_translator_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/core:input_batch_cache_test                            (cached) PASSED in 0.1s
>   WARNING: //dragnn/core:input_batch_cache_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/core:resource_container_test                           (cached) PASSED in 0.1s
>   WARNING: //dragnn/core:resource_container_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/io:sentence_input_batch_test                           (cached) PASSED in 0.1s
>   WARNING: //dragnn/io:sentence_input_batch_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/python:composite_optimizer_test                        (cached) PASSED in 4.3s
>   WARNING: //dragnn/python:composite_optimizer_test: Test execution time (4.3s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/python:digraph_ops_test                                (cached) PASSED in 2.3s
>   WARNING: //dragnn/python:digraph_ops_test: Test execution time (2.3s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/python:evaluation_test                                 (cached) PASSED in 2.2s
>   WARNING: //dragnn/python:evaluation_test: Test execution time (2.2s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/python:render_parse_tree_graphviz_test                 (cached) PASSED in 2.7s
>   WARNING: //dragnn/python:render_parse_tree_graphviz_test: Test execution time (2.7s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/python:render_spec_with_graphviz_test                  (cached) PASSED in 2.8s
>   WARNING: //dragnn/python:render_spec_with_graphviz_test: Test execution time (2.8s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/python:visualization_test                              (cached) PASSED in 7.9s
>   WARNING: //dragnn/python:visualization_test: Test execution time (7.9s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //syntaxnet:arc_standard_transitions_test                       (cached) PASSED in 0.1s
> //syntaxnet:binary_segment_state_test                           (cached) PASSED in 0.1s
> //syntaxnet:binary_segment_transitions_test                     (cached) PASSED in 0.1s
> //syntaxnet:char_ngram_string_extractor_test                    (cached) PASSED in 0.1s
>   WARNING: //syntaxnet:char_ngram_string_extractor_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //syntaxnet:char_properties_test                                (cached) PASSED in 0.1s
>   WARNING: //syntaxnet:char_properties_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //syntaxnet:char_shift_transitions_test                         (cached) PASSED in 0.1s
> //syntaxnet:head_transitions_test                               (cached) PASSED in 0.1s
> //syntaxnet:label_transitions_test                              (cached) PASSED in 0.1s
> //syntaxnet:morphology_label_set_test                           (cached) PASSED in 0.1s
>   WARNING: //syntaxnet:morphology_label_set_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //syntaxnet:once_transitions_test                               (cached) PASSED in 0.1s
> //syntaxnet:parser_features_test                                (cached) PASSED in 0.2s
> //syntaxnet:segmenter_utils_test                                (cached) PASSED in 0.1s
>   WARNING: //syntaxnet:segmenter_utils_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //syntaxnet:sentence_features_test                              (cached) PASSED in 0.1s
>   WARNING: //syntaxnet:sentence_features_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //syntaxnet:shared_store_test                                   (cached) PASSED in 0.1s
> //syntaxnet:tagger_transitions_test                             (cached) PASSED in 0.1s
> //syntaxnet/util:check_test                                     (cached) PASSED in 1.8s
>   WARNING: //syntaxnet/util:check_test: Test execution time (1.8s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //syntaxnet/util:registry_test                                  (cached) PASSED in 2.2s
>   WARNING: //syntaxnet/util:registry_test: Test execution time (2.2s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //syntaxnet:whole_sentence_features_test                        (cached) PASSED in 0.1s
>   WARNING: //syntaxnet:whole_sentence_features_test: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //util/utf8:unicodetext_unittest                                (cached) PASSED in 0.1s
>   WARNING: //util/utf8:unicodetext_unittest: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
> //dragnn/python:bulk_component_test                                      FAILED in 1.2s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/bulk_component_test/test.log
> //dragnn/python:graph_builder_test                                       FAILED in 1.9s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/graph_builder_test/test.log
> //dragnn/python:lexicon_test                                             FAILED in 1.2s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/lexicon_test/test.log
> //dragnn/python:network_units_test                                       FAILED in 1.2s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/network_units_test/test.log
> //dragnn/python:sentence_io_test                                         FAILED in 1.1s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/sentence_io_test/test.log
> //dragnn/python:spec_builder_test                                        FAILED in 1.2s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/spec_builder_test/test.log
> //dragnn/tools:model_trainer_test                                        FAILED in 1.3s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/tools/model_trainer_test/test.log
> //examples/dragnn:test_run_all_tutorials                                 FAILED in 1.0s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/examples/dragnn/test_run_all_tutorials/test.log
> //syntaxnet:beam_reader_ops_test                                         FAILED in 1.2s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/beam_reader_ops_test/test.log
> //syntaxnet:graph_builder_test                                           FAILED in 1.1s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/graph_builder_test/test.log
> //syntaxnet:lexicon_builder_test                                         FAILED in 0.8s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/lexicon_builder_test/test.log
> //syntaxnet:parser_trainer_test                                          FAILED in 1.8s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/parser_trainer_test/test.log
> //syntaxnet:reader_ops_test                                              FAILED in 1.3s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/reader_ops_test/test.log
> //syntaxnet:text_formats_test                                            FAILED in 1.2s
>   /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/text_formats_test/test.log

**Comment by SaintNazaire** [comment]
Introspected the 14 fails by reading tests logs in the container (please note that Syntaxnet indicates the error log subdirectory path as "/main/" whereas the actual subdirectory name is "/__main__/"):

**1.  //syntaxnet:text_formats_test**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/text_formats_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:04.758383: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x10efd28 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**2.  //dragnn/python:network_units_test**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/network_units_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:04.835508: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x2b6eda8 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**3.  //dragnn/python:bulk_component_test**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/bulk_component_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:04.853771: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x2457e88 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**4.  //syntaxnet:graph_builder_test**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/graph_builder_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:05.917530: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x244ffb8 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**5.  //examples/dragnn:test_run_all_tutorials** 

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/examples/dragnn/test_run_all_tutorials/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:05.915597: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x2524778 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/bin/examples/dragnn/test_run_all_tutorials.runfiles/__main__/examples/dragnn/test_run_all_tutorials: line 19: 14101 Aborted                 "$tutorial"

**6.  //syntaxnet:reader_ops_test**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/reader_ops_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:06.139938: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x29e0538 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**7.  //dragnn/python:spec_builder_test**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/spec_builder_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:07.114156: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x29380f8 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**8.  //dragnn/tools:model_trainer_test**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/tools/model_trainer_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:07.201428: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x1ee74c8 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/bin/dragnn/tools/model_trainer_test.runfiles/__main__/dragnn/tools/model_trainer_test: line 52: 14197 Aborted                 "${MODEL_TRAINER}" --model_dir="${TMP_MODEL_DIR}" --pretrain_steps='1' --train_epochs='10' --alsologtostderr

**9.  //dragnn/python:sentence_io_test** 

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/sentence_io_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:07.255833: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x158fd58 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**10.  //dragnn/python:graph_builder_test**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/graph_builder_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:08.502253: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x1faf658 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**11.  //syntaxnet:parser_trainer_test **

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/parser_trainer_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> + BINDIR=/root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/bin/syntaxnet/parser_trainer_test.runfiles/__main__/syntaxnet
> + CONTEXT=/root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/bin/syntaxnet/parser_trainer_test.runfiles/__main__/syntaxnet/testdata/context.pbtxt
> + TMP_DIR=/tmp/syntaxnet-output
> + mkdir -p /tmp/syntaxnet-output
> + sed s=OUTPATH=/tmp/syntaxnet-output=
> + sed s=SRCDIR=/root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/bin/syntaxnet/parser_trainer_test.runfiles= /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/bin/syntaxnet/parser_trainer_test.runfiles/__main__/syntaxnet/testdata/context.pbtxt
> + PARAMS=128-0.08-3600-0.9-0
> + /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/bin/syntaxnet/parser_trainer_test.runfiles/__main__/syntaxnet/parser_trainer --arg_prefix=brain_parser --batch_size=32 --compute_lexicon --decay_steps=3600 --graph_builder=greedy --hidden_layer_sizes=128 --learning_rate=0.08 --momentum=0.9 --output_path=/tmp/syntaxnet-output --task_context=/tmp/syntaxnet-output/context --training_corpus=training-corpus --tuning_corpus=tuning-corpus --params=128-0.08-3600-0.9-0 --num_epochs=12 --report_every=100 --checkpoint_every=1000 --logtostderr
> 2017-11-01 13:39:08.731766: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x2dcb4b8 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/bin/syntaxnet/parser_trainer_test.runfiles/__main__/syntaxnet/parser_trainer_test: line 52: 14328 Aborted                 "$BINDIR/parser_trainer" --arg_prefix=brain_parser --batch_size=32 --compute_lexicon --decay_steps=3600 --graph_builder=greedy --hidden_layer_sizes=128 --learning_rate=0.08 --momentum=0.9 --output_path=$TMP_DIR --task_context=$TMP_DIR/context --training_corpus=training-corpus --tuning_corpus=tuning-corpus --params=$PARAMS --num_epochs=12 --report_every=100 --checkpoint_every=1000 --logtostderr

**12.  //syntaxnet:lexicon_builder_test FAILED**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/lexicon_builder_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:09.884684: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x23776f8 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**13.  //syntaxnet:beam_reader_ops_test FAILED**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/syntaxnet/beam_reader_ops_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:10.212257: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x16d5198 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**14.  //dragnn/python:lexicon_test FAILED**

`cat /root/.cache/bazel/_bazel_root/3b4c7ccb85580bc382ce4a52e9580003/execroot/__main__/bazel-out/local-opt/testlogs/dragnn/python/lexicon_test/test.log`

> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> -----------------------------------------------------------------------------
> 2017-11-01 13:39:10.243809: F external/org_tensorflow/tensorflow/core/framework/variant_op_registry.cc:104] Check failed: existing == nullptr (0x1a2fc98 vs. nullptr)Unary VariantDecodeFn for type_name: tensorflow::Tensor already registered
> Aborted

**Comment by SaintNazaire** [comment]
Currently attempting to build without Tensorflow install

**Comment by SaintNazaire** [comment]
Dup of issues #2355 (also referenced in #2589) ?

**Comment by asimshankar** [comment]
Closing this, as it seems (like you pointed out) that this is a duplicate of #2355 and #2589

**Comment by SaintNazaire** [comment]
Could you please let know Tensorflow people that #2355 is quite a serious issue and is likely to be worth their attention? By the way cannot reproduce fixes in #2355, test fails increase from 14 to 22.
